llumnix/config/default.py

```diff
@@ -51,6 +51,70 @@
 # Disable redirecting all worker logs to driver
 _C.SERVER.DISABLE_LOG_TO_DRIVER = False
 
+# ----------------------------- V1 CONFIGURATION ------------------------------
```

Comment:
sorry for my change. i think we should remove this args, set default value in VLLMV1EntrypointArgs and remove from_llumnix_config function in VLLMV1EntrypointArgs.